src/Kirel.Identity.Exceptions/KirelNotFoundException.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Kirel.Identity.Exceptions;
+
+/// <summary>
+/// The exception that is thrown when the entity was not found
+/// </summary>
+[Serializable]
+public class KirelNotFoundException : Exception
+{
+    public KirelNotFoundException(){}
```

Comment:
You need to add a description to the constructors.

Reply:
done